### DISCUSSION TITLE
Release cadence

### DISCUSSION THREAD

**Discussion by daniv-msft** [buildpacks/community#223] [General]
Hello!
We are using the buildpack CLI in our product and are looking forward to a bug fix (https://github.com/buildpacks/pack/issues/1797).
Could you please clarify how often you release new versions?
Thanks again for all your work on this and help with the issues along the way!

CC: @cormacpayne

**Comment by natalieparellano** [comment]
Cross post from pack: https://github.com/buildpacks/pack/issues/1797#issuecomment-1622307280

**Comment by daniv-msft** [comment]
Thanks @natalieparellano! I appreciate your reply. I'm closing this one and we'll monitor the release of 0.30.0.
